# Rocket parameters
stages: 4
# negligible diameters -> neglect drag for the drone
diameters:
  - 1.0e-39
  - 1.0e-40
  - 1.0e-40
  - 1.0e-40
stage_z:
  - 0
  - 0.1
  - 0.2
  - 0.5
fin_n: 3
fin_top_offset: 0.25
fin_span: 1.0e-40
fin_root_chord: 0.1 # base length
fin_tip_chord: 0.1
fin_sweep: 0.1
fin_thickness: 0.004
lug_n: 2
lug_S: 1.0e-40
dry_mass: 1.533
dry_CM: 0.1
dry_I:
  - 0.0623
  - 0.0669
  - 0.0130
propellant_mass: 1.0e-40
propellant_CM: 0.1
#set Isp to -1 for constant propellant mass
Isp: -1

minPropellerSpeed: 35
maxPropellerSpeed: 65

maxServo1Angle: 0.1
maxServo2Angle: 0.1
CM_to_thrust_distance: 0.205
Cd:
  - 2
  - 2
  - 0.7
CM_offset_x: 0
initial_speed: 0